packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx

```diff
@@ -213,6 +213,20 @@ describe("GIVEN a Tooltip", () => {
     });
   });
 
+  describe("WHEN content is undefined", () => {
+    it("then tooltip doesn't display", () => {
```

Comment:
Can you also test "" please

Reply:
This test already exists - test above, line 208